src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_PARAMETER = 10.0;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random randomParameter = new Random();
+
+    public Figure getRandomFigure() {
+        switch (FigureName.values()[randomParameter.nextInt(FigureName.values().length)]) {

```

Comment:
Better to introduce `randomParameter.nextInt(FigureName.values().length)` as local variable